apps/fabric-website/src/components/ComponentPage/ComponentPage.scss

```diff
@@ -16,4 +16,8 @@
     @include contentPadding(50px, 50px);  // Override padding to match this website
   }
 
+  #Variants, #Implementation\ Examples, #Implementation {
```

Comment:
unnecessary code?

Reply:
This is a trickle-down from changes I had to make when we adopted the UHF. UHF was overwriting all of our h2 (and other typography) styles. So I had to use a more targeted selector to get our styles back. In the process, these more targeted selectors are now overwriting other styles we have. Thus the usage of ids in this case. In a previous PR, Angela mentioned that the spacing had gotten to large between sections, and this is one step to fix that.

Reply:
I understand the ID's, i'm curious what the `\ Examples` is for

Reply:
Escaping the space in 'Implementation Examples'

Reply:
oh funny....I never thought to have an ID with a space in it. is it programmatic, or can it be replaced with an '_'?

Reply:
the URL can have the _ while the label has a space

Reply:
We're using the id for in page nav - @battletoilet any thoughts?

Reply:
ID's are fine, I'm just saying to have the ID/href use _ instead of space. It's a small nit to make cleaner css

Reply:
Removed spaces!